### ISSUE TITLE
Missing element: <listPerson>

### ISSUE THREAD

**Issue by anacastrosalgado** [DARIAH-ERIC/lexicalresources#179]
Concerning the encoding of the introductory pages of MORAIS, I noticed that the TEI Lex-0 specification does not include the element `<listPerson>`. As I’m encoding the list of founders, this element is essential.

Examples:
 ``` xml
            <listPerson xml:lang="pt">
               <item>
                  <persName>Agoſtinho Joſé Martins Vidigal</persName>
                  <pc>,</pc>
                  <note>Medico</hi></note>
                  <pc>.</pc>
[...]
 ```

**Comment by laurentromary** [comment]
Notwithstanding the question on `<listPerson>`, I would avoid using `<item>` therein.

**Comment by daliboris** [comment]
`<item>` is not allowed within `<listPerson>` element, use `<person>` element instead.

See https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-listPerson.html

**Comment by anacastrosalgado** [comment]
Yes, if we use `<listPerson>`, I have to change `<item>` to  `<person>` . But probably we will not use it >>> @ttasovac

**Comment by ttasovac** [comment]
Decided not to allow listPerson, recommend list instead.
